### PR TITLE
feat: Added back button in see all reviews activity

### LINKED ISSUE #1952: Add back button in see all reviews activity
**Describe the feature you'd like**
There should be a back button in see all reviews activity.

**Additional context**

**Screenshots**

**Would you like to work on the issue?**
yes

### PR DESCRIPTION
**Pull request by chanmol1999**
Fixes #1952 Add back button in see all reviews activity
Changes:
Done a few additions in manifest file.
Screenshots for the change: 
screenshot_20190212_210515

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
Please do follow the pr guidelines :)

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -96,8 +96,11 @@
         <activity
             android:name=".skills.feedback.FeedbackActivity"
             android:label="See All Reviews"
-            android:parentActivityName=".skills.feedback.FeedbackActivity"
-            android:theme="@style/PreferencesThemeLight" />
+            android:theme="@style/PreferencesThemeLight">
+            <meta-data
+                android:name="android.support.PARENT_ACTIVITY"
+                android:value="org.fossasia.susi.ai.skills.SkillsActivity" />
+        </activity>
```
Comment (iamareebjamal) [inline app/src/main/AndroidManifest.xml]: Revert

Comment (chanmol1999) [reply app/src/main/AndroidManifest.xml]: Can you please elaborate.I am not able infer requested changes.

Comment (iamareebjamal) [reply app/src/main/AndroidManifest.xml]: Revert this change

Comment (chanmol1999) [reply app/src/main/AndroidManifest.xml]: Could you please review this @iamareebjamal

Comment (iamareebjamal) [reply app/src/main/AndroidManifest.xml]: I already did. Requested changes have still not been made

**Comment by chanmol1999** [comment]
@iamareebjamal sir I am not able to find a better way than this to implement back button.Please review @iamareebjamal @batbrain7

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (iamareebjamal) [reply app/src/main/AndroidManifest.xml]: Still not reverted

**Comment by iamareebjamal** [comment]
Revert all unneeded changes please

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -96,7 +96,7 @@
         <activity
             android:name=".skills.feedback.FeedbackActivity"
             android:label="See All Reviews"
-            android:theme="@style/PreferencesThemeLight" />
+            android:theme="@style/PreferencesThemeLight" />
```
Comment (immadisairaj) [inline app/src/main/AndroidManifest.xml]: There should not be any unnecessary spaces

**Pull request merged**